README.md

```diff
@@ -255,3 +255,68 @@ See the [metrics documentation](metrics).
 
 ## Kata Admission controller webhook
 See the [webhook documentation](kata-webhook).
+
+## Using Vagrant to test your code changes
+
+It is strongly recommended that you test your changes locally before opening
+the pull request as this can save people's time and CI resources. Because
+testing Kata Containers involve complex build and setup instructions, scripts
+on the .ci directory are created to easy and provide a reproducible process; but they
+are meant to run on CI environments that can be discarded after use. Therefore, some
+developers may find it dangerous to run those scripts on their workstations.
+
+That said, we provide on this repository a Vagrantfile which allows developers to use
```

Comment:
"Vagrantfile" isn't a known word so you'll either need to put it in backticks or add it as a word to the Kata dictionary, so it will be recognised as valid:

- https://github.com/kata-containers/tests/tree/main/cmd/check-spelling#adding-a-new-word